code/game/objects/items/devices/t_scanner.dm

```diff
@@ -10,53 +16,71 @@
 	matter = list(MATERIAL_PLASTIC = 2, MATERIAL_GLASS = 1)
 	origin_tech = list(TECH_MAGNET = 1, TECH_ENGINEERING = 1)
 
+
+	//Scan range can be changed, and the power costs scale up with it
 	var/scan_range = 1
 
-	var/on = 0
+	//TODO: Make devices have cell support as an inherent behaviour
+	var/obj/item/weapon/cell/cell = null
+	var/suitable_cell = /obj/item/weapon/cell/small
+	var/active_power_usage = 25 //Watts
+
+	var/turn_on_sound = 'sound/effects/Custom_flashlight.ogg'
+
+	/*Enabled and active are seperate things.
+	Enabled determines the power status. Is the scanner turned on or not?
+	The scanner is enabled as long as it has power, and the power switch is turned on. While enabled it will use power
+	*/
+	var/enabled = 0
```

Comment:
This is the only thing not according to our contributing, 1/0 instead of TRUE/FALSE